drip/drips.py

```diff
@@ -241,8 +251,10 @@ def get_count_from_queryset(self, MessageClass):
                     )
                     count += 1
             except Exception as e:
-                logging.error("Failed to send drip %s to user %s: %s" %
-                              (self.drip_model.id, user, e))
+                logging.error(
+                    "Failed to send drip %s to user %s: %s" %
+                    (self.drip_model.id, user, e)
```

Comment:
```suggestion
                    "Failed to send drip {} to user {}: {}".format(
                        self.drip_model.id,
                        user,
                        e,
                    )
```

Reply:
In this PR I just changed the indentation. There are lot of strings with `%`. I think it's better address this in other PR and solve it with a consistent solution

Reply:
Agree with @brunomichetti, I think we can leave that for later, otherwise it will become a huge PR